pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -911,6 +911,14 @@
     passthru.python3Dependencies = ps: [ ps.jupytext ];
   };
 
+  kulala-nvim = super.kulala-nvim.overrideAttrs {
+    buildInputs = [ curl ];
+    postPatch = ''
+      substituteInPlace lua/kulala/config/init.lua \
+        --replace 'curl_path = "curl"' 'curl_path = "${lib.getExe curl}"'
+    '';
```

Comment:
Isn't there the new "passtrhu/snippet" for declaring those opt-in patches ?
Or maybe it doesn't apply here.

Reply:
There is (https://github.com/NixOS/nixpkgs/pull/334913), but does it apply to `.setup` functions?

https://github.com/mistweaverco/kulala.nvim/blob/f6bbeeff6f8eaa2dc9b09148b6e0746ced5951ca/lua/kulala/config/init.lua#L93

Also, it is hard to find, so I would expect false bug reports about it

Reply:
It is hard to add passthru here, it config like
```lua
require("kulala").setup({
})
```

Reply:
Someone use other way manager neovim plugins, just use pkgs without other config. And, I think this passthru is useful to vim.g.xxx.
If curl path setted by vim.g.curl_path, passthru is the best way.